pkg/chart/chart_test.go

```diff
@@ -96,3 +96,66 @@ func TestMetadata(t *testing.T) {
 	is.Equal("1.0.0", chrt.AppVersion())
 	is.Equal(nil, chrt.Validate())
 }
+
+func TestIsRoot(t *testing.T) {
+	chrt1 := Chart{
+		parent: &Chart{
+			Metadata: &Metadata{
+				Name: "foo.yaml",
+			},
+		},
+	}
+
+	chrt2 := Chart{
+		Metadata: &Metadata{
+			Name: "foo.yaml",
+		},
+	}
+
+	is := assert.New(t)
+
+	is.Equal(false, chrt1.IsRoot())
+	is.Equal(true, chrt2.IsRoot())
+}
+
+func TestChartPath(t *testing.T) {
+	chrt1 := Chart{
+		parent: &Chart{
+			Metadata: &Metadata{
+				Name: "foo.yaml",
+			},
+		},
+	}
+
+	chrt2 := Chart{
+		Metadata: &Metadata{
+			Name: "foo.yaml",
+		},
+	}
+
+	is := assert.New(t)
+
+	is.Equal("foo.yaml.", chrt1.ChartPath())
+	is.Equal("foo.yaml", chrt2.ChartPath())
+}
+
+func TestChartFullPath(t *testing.T) {
+	chrt1 := Chart{
+		parent: &Chart{
+			Metadata: &Metadata{
+				Name: "foo.yaml",
+			},
+		},
+	}
+
+	chrt2 := Chart{
+		Metadata: &Metadata{
+			Name: "foo.yaml",
+		},
+	}
+
+	is := assert.New(t)
+
+	is.Equal("foo.yaml/charts/", chrt1.ChartFullPath())
+	is.Equal("foo.yaml", chrt2.ChartFullPath())
```

Comment:
can you remove the .yaml extension from the chart name?

Reply:
updated, PTAL